easybuild/easyblocks/i/icc.py

```diff
@@ -38,9 +38,9 @@
 import re
 from easybuild.tools import LooseVersion
 
-from easybuild.easyblocks.generic.intelbase import IntelBase, ACTIVATION_NAME_2012, COMP_ALL
-from easybuild.easyblocks.generic.intelbase import LICENSE_FILE_NAME_2012
+from easybuild.easyblocks.generic.intelbase import IntelBase, COMP_ALL
 from easybuild.easyblocks.t.tbb import get_tbb_gccprefix
```

Comment:
```suggestion
from easybuild.easyblocks.tbb import get_tbb_gccprefix
```

Though I think this fixes the issue @Micket and I have been seeing. Not sure if we need to fix this first, before this PR though.

Reply:
see #3543